html/elements/input/color.json

```diff
@@ -36,7 +36,7 @@
                 "version_added": true
               },
               "safari": {
-                "version_added": "10"
+                "version_added": "12.1"
               },
               "safari_ios": {
                 "version_added": false
```

Comment:
Safari for iOS should also be marked as supporting this (it's called 12.2 because that's the iOS version):

```suggestion
                "version_added": "12.2"
```

Tested on my iPhone running iOS 12.2:
![image](https://user-images.githubusercontent.com/2977353/55772003-9b86ee80-5a47-11e9-8ae1-65b7dba24ba6.png)